lib/utils/config.go

```diff
@@ -18,20 +18,22 @@ package utils
 
 import (
 	"os"
+	"path/filepath"
 	"strings"
 
 	"github.com/gravitational/trace"
 )
 
-// ReadToken is a utility function to read the token
-// from the disk if it looks like a path,
-// otherwise, treat it as a value
-func ReadToken(token string) (string, error) {
-	if !strings.HasPrefix(token, "/") {
-		return token, nil
+// TryReadValueAsFile is a utility function to read a value
+// from the disk if it looks like an absolute path,
+// otherwise, treat it as a value.
+// It only support absolute paths to avoid ambiguity in interpretation of the value
+func TryReadValueAsFile(value string) (string, error) {
+	if !filepath.IsAbs(value) {
+		return value, nil
 	}
-	// treat it as a file
-	out, err := os.ReadFile(token)
+	// treat it as an absolute filepath
+	out, err := os.ReadFile(value)
```

Comment:
If the user has multiple CA pins and wants them in a file rather than the teleport config, IMO it would be nice to allow multiple pins in the same file rather than requiring a separate file for each pin

Reply:
I like that suggestion and the same would apply to tokens

Reply:
I believe you can only have a single `auth_token`, though

Reply:
You're right, I was confusing myself with `auth_service.tokens`

Reply:
Oh right, yeah I think that is fine, it should be backward compatible. I think those can include commas, right? Probably best to just split on whitespace then

Reply:
The config for those looks like either of these:
```
auth_service:
    tokens:
        - node:some-static-token
        - node,proxy:/Some/Path/To/Token/File
```

So the file is the same format.

If we do allow multiple pins in a file, should they just be newline separated? That would seem the most obvious way to me.

Reply:
Yeah `bufio.Scanner` splits by line by default, might as well do that

Reply:
I kept it more generic and just have the TryReadValueAsFile function return a string. Then on caller side I use strings.Split